roles/openshift_logging/README.md

```diff
@@ -57,6 +57,8 @@ When both `openshift_logging_install_logging` and `openshift_logging_upgrade_log
 - `openshift_logging_fluentd_hosts`: List of nodes that should be labeled for Fluentd to be deployed to. Defaults to ['--all'].
 - `openshift_logging_fluentd_buffer_queue_limit`: Buffer queue limit for Fluentd. Defaults to 1024.
 - `openshift_logging_fluentd_buffer_size_limit`: Buffer chunk limit for Fluentd. Defaults to 1m.
+- `openshift_logging_fluentd_buffer_type`: Fluentd will set the buffer type.  Defaults to 'file'.
```

Comment:
Having the ability to deploy with memory buffer, even if just for testing/PoC, is useful.  Let's keep this knob, and default it to file.